app/src/main/java/com/example/touchcar/presentation/main_menu/MainMenuFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.touchcar.presentation.main_menu
+
+import android.graphics.Insets.add
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.FragmentActivity
+import androidx.fragment.app.add
+import androidx.fragment.app.commit
+import androidx.recyclerview.widget.RecyclerView
+import com.example.touchcar.R
+import com.example.touchcar.domain.entity.Manufacturer
+import com.example.touchcar.presentation.main_menu.bottom_sheet.BottomSheetFragment
+import com.example.touchcar.presentation.main_menu.recycler.MainMenuAdapter
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+
+
+@AndroidEntryPoint
+class MainMenuFragment : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+    private lateinit var mainMenuAdapter: MainMenuAdapter
+    private lateinit var recyclerView: RecyclerView
+
+    @Inject
+    lateinit var mainMenuViewModel: MainMenuViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_main_menu, container, false)
+    }
+
+    override fun onViewCreated(itemView: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(itemView, savedInstanceState)
+
+        mainMenuViewModel.manufacturerLiveData
+            .observe(
+                this, { manufacturers ->
+                    mainMenuAdapter.manufacturers = manufacturers
+                })
```

Comment:
Давай это в более читаемый вид приведём:
```suggestion
        mainMenuViewModel.manufacturerLiveData.observe(this) { manufacturers ->
            mainMenuAdapter.manufacturers = manufacturers
        }
```